src/AggregateFunctions/AggregateFunctionArgMinMax.h

```diff
@@ -89,6 +90,8 @@ class AggregateFunctionArgMinMax final : public IAggregateFunctionDataHelper<Dat
     {
         this->data(place).result.read(buf, *serialization_res, arena);
         this->data(place).value.read(buf, *serialization_val, arena);
+        if (unlikely(this->data(place).value.has() && !this->data(place).result.has()))
```

Comment:
What about the other way around? It should be possible to do the same in the other order, right?

Reply:
Yes, but seems like we don't care about `result` if there's no `value`. If `this->data(place).value.has()` is `flase`, then we will never try to access `result`. The issue is that we could try to access invalid `result` when `value` is valid.

Reply:
A reverse example would be:
```
SELECT 'fuzz2', finalizeAggregation(CAST(unhex('0100000000000000000FFFFFFFF0'), 'AggregateFunction(argMax, UInt64, String)'));
```

I'm going to add it as a test and also catch it